Mastodon/Diffiable/Profile/ProfileFieldSection.swift

```diff
@@ -57,7 +58,17 @@ extension ProfileFieldSection {
             // set background
             var backgroundConfiguration = UIBackgroundConfiguration.listPlainCell()
             backgroundConfiguration.backgroundColor = UIColor.secondarySystemBackground
+            if (field.verifiedAt.value != nil) {
+                backgroundConfiguration.backgroundColor = Asset.Scene.Profile.About.bioAboutFieldValidatedBackground.color
+            }
             cell.backgroundConfiguration = backgroundConfiguration
+            
+            // set checkmark
+            cell.checkmark.isHidden = true
+            if let verifiedAt = field.verifiedAt.value {
+                cell.checkmark.isHidden = false
+                cell.checkmark.accessibilityLabel = "Ownership of this link was checked on \(verifiedAt)" // TODO: I18N / L10N
```

Comment:
It would be cool to surface this for folks who don’t use accessibility tools

Reply:
I agree, but I'm not sure how best to display it. A few other mastodon clients I tried only show it in voiceover, and the web-app relies on the 'title' attribute which doesn't work with touch input.

Do you have any suggestions?

Reply:
I've gone with a simple UIAlertController when you tap the checkmark, for now:
<img src="https://user-images.githubusercontent.com/1376403/201451052-bb673e82-e656-4bf6-9470-5d64222625b2.png" height="400" />

Reply:
I realised I'd been using the wrong term; I've updated all variants of 'Validated' to 'Verified'.

Reply:
Another option would be to use the edit menu APIs (for example, try tapping a timestamp in the GitHub iOS app)

Reply:
OK, that has been re-implemented with UIMenuController and UIEditMenuInteraction depending on the OS version. It does mean it can be contained in the ProfileFieldCollectionViewCell, which is nice :)

I've had to shorten the text so it doesn't get truncated, but it does expose the time of verification now.

<img src="https://user-images.githubusercontent.com/1376403/201486631-bfa37e4f-96e4-45e7-91fc-ee133946b73f.png" height="400" />